code/modules/reagents/reagent_containers/whipped_canister.dm

```diff
@@ -0,0 +1,78 @@
+/obj/item/reagent_containers/food/whipped_canister // For when you want your cakes and pies to have a little extra something (bath salts)
```

Comment:
This should probably be a subtype of `/obj/item/reagent_containers/food/condiment`, saving you a lot of code duplication and making it easier for whoever wants to change these later.

Reply:
Don't condiment containers change icons depending on what reagent is inside of them?

Reply:
No, those are packs. `/obj/item/reagent_containers/food/condiment/pack`

`/obj/item/reagent_containers/food/condiment` is just a generic reagent container with a different attack, afterattack, suicide_act, and update_icon_state.